src/audio/commands/Audio/exportqueue.ts

```diff
@@ -14,8 +16,8 @@ export const maximumExportQueueSize = 100;
 	cooldownLimit: 10,
 	description: LanguageKeys.Commands.Music.ExportQueueDescription,
 	extendedHelp: LanguageKeys.Commands.Music.ExportQueueExtended,
-	requiredClientPermissions: ['ATTACH_FILES'],
-	runIn: ['GUILD_ANY']
+	requiredClientPermissions: [PermissionFlags.ATTACH_FILES],
+	runIn: [CommandOptionsRunTypeEnum.GuildAny]
```

Comment:
Holy shit why did we suffix it with Enum in sapphire...

Reply:
Because `CommandOptionsRunType` is the type of the string union